core/domain/exp_domain.py

```diff
@@ -550,6 +552,8 @@ def __init__(self, version, state_interaction_ids_dict):
 class Exploration:
     """Domain object for an Oppia exploration."""
 
+    __initialized = False
```

Comment:
Why is this needed? There should, at the very least, be a clear comment explaining it.

Also, why are you putting double underscores in front of it? That is a convention reserved only for internal Python methods.

Lastly I think this might not be what you're going for. If you want to know whether an instance is initialized, you can set something like self.initialized (or there might be another way). But here you seem to be declaring it on the exploration _class_.

Reply:
Removed, this was used to block the initialization part, but removed now as not needed any more.